processing/src/main/java/org/apache/carbondata/processing/loading/steps/InputProcessorStepWithNoConverterImpl.java

```diff
@@ -334,6 +356,42 @@ private CarbonRowBatch getBatch() {
       return newData;
     }
 
+    private Object[] convertToNoDictionaryToBytesWithoutReArrange(Object[] data,
```

Comment:
This method is almost same as 'convertToNoDictionaryToBytes'. Better to use the withoutReArrange to skip the rearrange in that method.

Reply:
I checked , This is different. In original method when ordered data is present it is based on no dictionary mapping, I don't want to use no dictionary mapping.
complex type logic is same, may be I will extract a common method for that